sungc/interactive.py

```diff
@@ -43,12 +47,35 @@ class ROI_Selector(object):
           line connecting two existing vertices
     """
 
-    def __init__(self, ax=None, poly_xy=None, max_ds=10):
+    def __init__(
+        self,
+        ax: matplotlib.axes.Axes,
+        poly_xy: Union[list, tuple, None] = None,
```

Comment:
Have a look at [mypy docs](https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html#built-in-types) on how to use typing hints.

Reply:
Might be easiest to look at a future PR to implement typing hints.